bootstrap/bootstrap.go

```diff
@@ -431,11 +432,42 @@ func (b *Bootstrap) PluginPhase() error {
 		checkout, err := b.checkoutPlugin(p)
 		if err != nil {
 			return errors.Wrapf(err, "Failed to checkout plugin %s", p.Name())
-
+		}
+		if b.Config.PluginValidation && checkout.Definition == nil {
+			if b.Debug {
+				b.shell.Commentf("Parsing plugin definition for %q from %s", p.Label(), checkout.CheckoutDir)
+			}
+			// parse the plugin definition from the plugin checkout dir
+			checkout.Definition, err = plugin.LoadDefinitionFromDir(checkout.CheckoutDir)
+			if err == plugin.ErrDefinitionNotFound {
+				b.shell.Warningf("Failed to find plugin definition")
```

Comment:
Should this include the name of the plugin? Or is the name of the plugin already shown?

Reply:
(Based on the error here: https://github.com/buildkite/agent/pull/748/files#diff-1d1158568615ba7f7c717c11183449f8L433)

Reply:
Fixed!